conf/main/logback.xml

```diff
@@ -35,7 +35,7 @@
     </logger>
 
     <!-- It is possible to change the logging level to a more granular level. For example:-->
-    <!--<logger name="io.mindmaps.engine.postprocessing.BackgroundTasks" level="DEBUG">-->
+    <!--<logger name="io.mindmaps.engine.postprocessing.PostProcessing" level="DEBUG">-->
```

Comment:
Should probably clean up these comments. I was confused with the name change considering that the name is now `BackgroundTasks`